### PR TITLE
[db-sync] Add the `d_b_workspace_cluster` table to sync config

### PR DESCRIPTION
**Pull request by andrew-farries**
## Description

Add the `d_b_workspace_cluster` table to the `db-sync` config to start syncing the table.

<details>
<summary>Context </summary>
As part of #9198  we want to start syncing the `d_b_workspace_cluster` table with `db-sync`.

Currently, the table differs between US and EU regions because each table contains only the data relevant to that region. For example, in the EU table, the `eu70` workspace cluster is marked as `available` and the `us70` cluster is `cordoned`.  In the US cluster `eu70` is `cordoned` and `us70` is `available`.

In order to sync the table we need to get to a point where there is no difference in the data in the table between EU and US regions.

To do that we will introduce a new field in the table called `applicationCluster` which records the name of the application cluster to which the record belongs. Thus, for each workspace cluster there will be two rows in Gitpod SaaS:

| name | applicationCluster | url | tls | state | ... |
| --- | --- | --- | --- | --- | --- |
| eu70 | eu02 | url | tls info | available | ... |
| eu70 | us02 | url | tls info | cordoned | ... |

Effectively the new `applicationCluster` column gives the table an extra dimension so that we can combine both tables (EU and US) into one.

#13722  added the column to the table and made `gpctl` fill the value when `gpctl register`ing a new workspace cluster. The value is taken from the `GITPOD_INSTALLATION_SHORTNAME` environment variable in `ws-manager-bridge`.
</details>

## Related Issue(s)

Part of #9198 and #13800 

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`

### CODE REVIEW

**Comment by geropl** [comment]
/hold

Depends on: https://github.com/gitpod-io/gitpod/pull/14134#issuecomment-1290053120

Also, did you announce the  rollout to team workspace? It should not affect them. But they should be aware of the impact, etc.

**Review by easyCZ — APPROVED** [review]
LGTM.

There's a hold from @geropl around follow-up.

**Comment by andrew-farries** [comment]
> /hold
> 
> Depends on: [#14134 (comment)](https://github.com/gitpod-io/gitpod/pull/14134#issuecomment-1290053120)
> 
> Also, did you announce the rollout to team workspace? It should not affect them. But they should be aware of the impact, etc.

Yes, that was done [here (internal Slack)](https://gitpod.slack.com/archives/C02F19UUW6S/p1666699617578319).

**Comment by geropl** [comment]
We just realized that before merging this we need to switch the table to the "soft deletion" pattern.

File: components/gitpod-db/src/tables.ts
Comment (geropl) [inline components/gitpod-db/src/tables.ts:54]: :+1:

**Pull request merged**
